interpreter.go

```diff
@@ -422,19 +311,61 @@ func (i *interpreter) evaluate(a astNode) (value, error) {
 				// Omitted field.
 				continue
 			default:
-				return nil, makeRuntimeError(ast.Loc(), "Field name was not a string.")
+				return nil, e.Error("Field name was not a string.")
 			}
 
 			if err != nil {
```

Comment:
this is already checked, above

Reply:
(test is already pushed `testdata/fieldname_not_string.input`)